Makefile

```diff
@@ -106,13 +106,21 @@ generate: controller-gen ## Generate code containing DeepCopy, DeepCopyInto, and
 	$(CONTROLLER_GEN) object:headerFile="hack/boilerplate.go.txt" paths="./..."
 
 .PHONY: fmt
-fmt: goimports ## Format the code using goimports
-	find . -not -path '*/\.*' -name '*.go' -exec $(GOIMPORTS) -w {} \;
+fmt: ## Run go fmt verifications
+	hack/verify-gofmt.sh
```

Comment:
I used to run `make fmt` to have the code formatted, but with this, it would just verify and exit, which is kinda surprising to me. If we are just calling `./hack/verify-gofmt.sh`, I would suggest renaming this Make target into `verify-fmt` to make the intent clearer..

I noticed that `goio` also formats the code as well, besides organizing the imports. So maybe this target could just be this instead:
```
fmt: imports ## Format the code using goio
```

Then a separate `verify-fmt` target that would run `hack/verify-gofmt.sh`.

WDYT?

Reply:
Do we need fmt verification at all?

Reply:
@coreydaley WDYT?